### PR TITLE
OSDOCS-2471: Update capabilities list for SR-IOV DPDK and RDMA examples

### PR DESCRIPTION
**Pull request by jboxman**
- https://issues.redhat.com/browse/OSDOCS-2471

This updates the examples with some required capabilities. Applies to all versions.

Preview: https://deploy-preview-35934--osdocs.netlify.app/openshift-enterprise/latest/networking/hardware_networks/using-dpdk-and-rdma

### CODE REVIEW

**Comment by jboxman** [comment]
@EinatGlottmann, I've created this PR for the changes that apply to all releases.

File: modules/nw-sriov-dpdk-example-intel.adoc [hunk 1]
```diff
@@ -101,7 +101,7 @@ spec:
     image: <DPDK_image> <2>
     securityContext:
      capabilities:
```
Comment (EinatGlottmann) [inline modules/nw-sriov-dpdk-example-intel.adoc:103]: Please add "runAsUser: 0" under the securityContext

securityContext structure should look as follows: 
securityContext:
  runAsUser: 0
  capabilities:
    add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"]

File: modules/nw-sriov-dpdk-example-mellanox.adoc [hunk 2]
```diff
@@ -105,7 +105,7 @@ spec:
     image: <DPDK_image> <2>
     securityContext:
```
Comment (EinatGlottmann) [inline modules/nw-sriov-dpdk-example-mellanox.adoc:106]: Please add "runAsUser: 0" under the securityContext

securityContext structure should look as follows: 
securityContext:
  runAsUser: 0
  capabilities:
    add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"]

File: modules/nw-sriov-rdma-example-mellanox.adoc [hunk 3]
```diff
@@ -108,7 +108,7 @@ spec:
     image: <RDMA_image> <2>
     securityContext:
```
Comment (EinatGlottmann) [inline modules/nw-sriov-rdma-example-mellanox.adoc:109]: Please add "runAsUser: 0" under the securityContext

securityContext structure should look as follows: 
securityContext:
  runAsUser: 0
  capabilities:
    add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"]

**Comment by EinatGlottmann** [comment]
@jboxman, please also update the securityContext in the following page:
https://docs.openshift.com/container-platform/4.8/networking/hardware_networks/about-sriov.html#example-vf-use-in-pod_about-sriov

**Comment by jboxman** [comment]
@EinatGlottmann, updated!

**Review by EinatGlottmann — CHANGES_REQUESTED** [review]
@jboxman, please update ipam config example 
https://github.com/openshift/openshift-docs/pull/35934/files#diff-4046d70daeba3eb42605e7d700c3cb9c3839d89d5b9cbd386fb5389e5e99fdc7R69 
to be the same as in the Mellanox NIC example in the document (it is the same as the correction you made here: https://github.com/openshift/openshift-docs/pull/35276)

**Review by EinatGlottmann — CHANGES_REQUESTED** [review]
@jboxman, please update the security context here as well (same link, line 48) 
https://github.com/openshift/openshift-docs/pull/35934/commits/a9d5a085fe1b5209343207f129466ac0a84f4728#diff-b654e3eca41e31ffaf2fd78074cd05a34d2326c0c1b91d84cf2f4136f838812eR48

**Comment by jboxman** [comment]
> @jboxman, please update the security context here as well (same link, line 48)
> [a9d5a08#diff-b654e3eca41e31ffaf2fd78074cd05a34d2326c0c1b91d84cf2f4136f838812eR48](https://github.com/openshift/openshift-docs/commit/a9d5a085fe1b5209343207f129466ac0a84f4728#diff-b654e3eca41e31ffaf2fd78074cd05a34d2326c0c1b91d84cf2f4136f838812eR48)

I have this added in all 4 places now:

```
modules/nw-sriov-dpdk-example-intel.adoc:
  105        capabilities:
  106:         add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"] <3>
  107      volumeMounts:

modules/nw-sriov-dpdk-example-mellanox.adoc:
  108        capabilities:
  109:         add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"] <3>
  110      volumeMounts:

modules/nw-sriov-example-vf-function-in-pod.adoc:
  28        capabilities:
  29:         add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"]
  30      command: ["sleep", "infinity"]

modules/nw-sriov-rdma-example-mellanox.adoc:
  111        capabilities:
  112:         add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"] <3>
  113      volumeMounts:
```

Is that what you mean?

Thanks!

**Comment by EinatGlottmann** [comment]
There is 1 more place: 
modules/nw-sriov-example-vf-function-in-pod.adoc  in line 48

**Comment by jboxman** [comment]
@EinatGlottmann added

**Review by EinatGlottmann — APPROVED** [review]
LGTM

**Comment by jboxman** [comment]
@zhaozhanqi, this PR adds changes that apply to all OCP versions; Does this look okay? Thanks!

**Comment by zhaozhanqi** [comment]
ok, /lgtm

**Pull request merged**